### PR TITLE
kola/docker: add selinux test

### PR DESCRIPTION
**Pull request by tormath1**
we create a simple scenario where the user would run a docker container
with a shared volume within SELinux context.

## How to use

:warning: can't be run with current Flatcar release as it needs SELinux updates

## Testing done

```
sudo ./bin/kola run -b cl -p qemu --qemu-image flatcar_production_qemu_image.img docker.selinux
```

closes https://github.com/kinvolk/Flatcar/issues/304

### CODE REVIEW

**Comment by tormath1** [comment]
@kinvolk/flatcar-maintainers if you have a few minutes to review this - it includes a basic SELinux test for Docker. :) I force pushed to include `beta` to supported versions.

**Review by jepio — APPROVED** [review]
Looks good. Would it be possible to directly skip on arm64 as long as we don't have selinux there?

**Comment by tormath1** [comment]
@jepio thanks for the review ! `arm64` is now excluded from the test. Let's wait for the CI, then squash and merge.

**Pull request merged**
